src/Http/Action/Security/SsoAction.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Security;
+
+use OpenCFP\Domain\Services;
+use Symfony\Component\HttpFoundation;
+use Symfony\Component\Routing;
+
+final class SsoAction
+{
+    /**
+     * @var Services\Authentication
+     */
+    private $authentication;
+
+    /** @var int */
+    private $clientId;
+
+    /** @var string */
+    private $redirectUri;
+
+    /** @var string */
+    private $authorizeUrl;
+
+    /**
+     * @var Routing\Generator\UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    /** @var string */
+    private $sso;
+
+    /**
+     * LoginAction constructor.
+     *
+     * @param Services\Authentication                 $authentication
+     * @param Routing\Generator\UrlGeneratorInterface $urlGenerator
+     * @param int                                     $clientId
+     * @param string                                  $redirectUri
+     * @param string                                  $authorizeUrl
+     */
```

Comment:
What do you think about removing the DocBlock, as it only repeats information that is already present from the type declarations of the method arguments? Also, following the rename of the action, the short description has become misleading.